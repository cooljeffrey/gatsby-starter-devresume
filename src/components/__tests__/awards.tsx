import React from "react";
import renderer from "react-test-renderer";
import Awards from "../awards";
describe("Awards", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Awards items={[{ name: "Award", organization: "Org" }]} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
