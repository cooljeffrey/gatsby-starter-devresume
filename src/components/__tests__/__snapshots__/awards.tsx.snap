// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Awards renders correctly 1`] = `
<section
  className="education-section py-3"
>
  <h3
    className="text-uppercase resume-section-heading mb-4"
  >
    Awards
  </h3>
  <ul
    className="list-unstyled resume-awards-list"
  >
    <li
      className="mb-3"
    >
      <div
        className="font-weight-bold"
      >
        Award
      </div>
      <div
        className="text-muted"
      >
        Org
      </div>
    </li>
  </ul>
</section>
`;
